src/simulation/LoadGenerator.cpp

```diff
@@ -2183,9 +2220,7 @@ LoadGenerator::execute(TransactionTestFramePtr& txf, LoadGenMode mode,
                 : "";
         CLOG_INFO(LoadGen, "tx rejected '{}': ===> {}, {}",
                   TX_STATUS_STRING[static_cast<int>(addResult.code)],
-                  txf->isSoroban() ? "soroban"
-                                   : xdrToCerealString(txf->getEnvelope(),
-                                                       "TransactionEnvelope"),
+                  xdrToCerealString(txf->getEnvelope(), "TransactionEnvelope"),
```

Comment:
I think the problem is with transactions like UPLOAD_WASM, which print the whole multi-KB WASM blob. Ideally, we print the envelope contents without WASM, but that seems outside of the scope of this PR, so I'd just revert this.